argus/_version.py

```diff
@@ -0,0 +1,21 @@
+# file generated by setuptools-scm
+# don't change, don't track in version control
```

Comment:
add this file to gitignore and un-commit it

Reply:
I thought I've add it to .gitignore, but seems like I've forgot about it